.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
```

Comment:
I would like all GH workflows to end with yaml, but according to the official documentation, the file should be called dependabot.yml.

Reply:
I just ran the wizard...